cmd/authenticator/main.go

```diff
@@ -10,6 +11,7 @@ import (
 	"github.com/cyberark/conjur-authn-k8s-client/pkg/authenticator"
 	authnConfig "github.com/cyberark/conjur-authn-k8s-client/pkg/authenticator/config"
 	"github.com/cyberark/conjur-authn-k8s-client/pkg/log"
+	"github.com/cyberark/secrets-provider-for-k8s/pkg/trace"
```

Comment:
Is it bad practice to have a dependency on the cyberark/secrets-provider-for-k8s repo, which has packages that depend on this repo?